src/XMakeCommandLine/Microsoft.Build.CommonTypes.xsd

```diff
@@ -213,6 +213,29 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="DotNetCliToolReference" substitutionGroup="msb:Item">
+    <xs:annotation>
+      <xs:documentation>
+        <!-- _locID_text="DotNetCliToolReference" _locComment="" -->A project-specific tool that can be ran from the command line. 
+      </xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
+      <xs:complexContent>
+        <xs:extension base="msb:SimpleItemType">
+          <xs:sequence>
+            <xs:element name="Version" />
+          </xs:sequence>
+          <xs:attribute name="Include" type="xs:string">
+            <xs:annotation>
+              <xs:documentation>
+                <!-- _locID_text="DotNetCliToolReference_Include" _locComment="" -->Package name of the tool. This may differ from its associated reference package name.
```

Comment:
I'm not sure I understand your comment but I'm replacing that with the public documentation for this tag:

"The CLI tool that the user wants restored in the context of the project."